HISTORY.md

```diff
@@ -133,6 +137,7 @@
   * The DecisionStump class has been marked deprecated; use the `DecisionTree`
     class with `NoRecursion=true` or use `ID3DecisionStump` instead (#2099).
 
+>>>>>>> origin/master
```

Comment:
```suggestion
```

Reply:
What?  How did this get here.  I thought I removed it?  Anyway, nice catch. :)